From 5bfd0c020fa14b89f1522ff257f47e92fb3d1136 Mon Sep 17 00:00:00 2001
From: Stanislaw Gruszka <sgruszka@redhat.com>
Date: Wed, 10 Dec 2014 10:29:31 -0500
Subject: [wireless] rt2x00: add new rt2800usb devices

Message-id: <1418207389-7899-18-git-send-email-sgruszka@redhat.com>
Patchwork-id: 101588
O-Subject: [RHEL7.1 PATCH 17/35] wireless: rt2x00: add new rt2800usb devices
Bugzilla: 1155538
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: John Greene <jogreene@redhat.com>

From: Xose Vazquez Perez <xose.vazquez@gmail.com>

BZ#1155538

commit 6a06e554daef86c4e8d290284927b081fedb249e upstream.

0x0b05 0x17e8 RT5372 USB 2.0  bgn 2x2 ASUS USB-N14
0x0411 0x0253 RT5572 USB 2.0 abgn 2x2 BUFFALO WLP-U2-300D
0x0df6 0x0078 RT???? Sitecom N300

Cc: Ivo van Doorn <IvDoorn@gmail.com>
Cc: Helmut Schaa <helmut.schaa@googlemail.com>
Cc: John W. Linville <linville@tuxdriver.com>
Cc: users@rt2x00.serialmonkey.com
Cc: linux-wireless@vger.kernel.org
Signed-off-by: Xose Vazquez Perez <xose.vazquez@gmail.com>
Signed-off-by: John W. Linville <linville@tuxdriver.com>
Cc: Stanislaw Gruszka <sgruszka@redhat.com>
Signed-off-by: Luis Henriques <luis.henriques@canonical.com>

Brew:
https://brewweb.devel.redhat.com/taskinfo?taskID=8330268

Testing:
Compiled only.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/wireless/rt2x00/rt2800usb.c b/drivers/net/wireless/rt2x00/rt2800usb.c
index 48bdb87..8e00ffa 100644
--- a/drivers/net/wireless/rt2x00/rt2800usb.c
+++ b/drivers/net/wireless/rt2x00/rt2800usb.c
@@ -1284,6 +1284,8 @@ static struct usb_device_id rt2800usb_device_table[] = {
  /* Arcadyan */
  { USB_DEVICE(0x043e, 0x7a12) },
  { USB_DEVICE(0x043e, 0x7a32) },
+ /* ASUS */
+ { USB_DEVICE(0x0b05, 0x17e8) },
  /* Azurewave */
  { USB_DEVICE(0x13d3, 0x3329) },
  { USB_DEVICE(0x13d3, 0x3365) },
@@ -1320,6 +1322,7 @@ static struct usb_device_id rt2800usb_device_table[] = {
  { USB_DEVICE(0x057c, 0x8501) },
  /* Buffalo */
  { USB_DEVICE(0x0411, 0x0241) },
+ { USB_DEVICE(0x0411, 0x0253) },
  /* D-Link */
  { USB_DEVICE(0x2001, 0x3c1a) },
  { USB_DEVICE(0x2001, 0x3c21) },
@@ -1410,6 +1413,7 @@ static struct usb_device_id rt2800usb_device_table[] = {
  { USB_DEVICE(0x0df6, 0x0053) },
  { USB_DEVICE(0x0df6, 0x0069) },
  { USB_DEVICE(0x0df6, 0x006f) },
+ { USB_DEVICE(0x0df6, 0x0078) },
  /* SMC */
  { USB_DEVICE(0x083a, 0xa512) },
  { USB_DEVICE(0x083a, 0xc522) },
-- 
1.7.1